app/src/main/java/chat/rocket/android/authentication/login/presentation/LoginPresenter.kt

```diff
@@ -50,7 +50,7 @@ class LoginPresenter @Inject constructor(
     val serverInteractor: GetConnectingServerInteractor
 ) {
     // TODO - we should validate the current server when opening the app, and have a nonnull get()
-    private var currentServer = serverInteractor.get()!!
+    private var currentServer = serverInteractor.get()?: "https://open.rocket.chat"
```

Comment:
Why add the default server url here?

Reply:
Without this, the tests would fail as current server is becoming null.  The app should validate the current server when opening the app and have a non-null get, there is also a todo regarding this.

Reply:
Just so that I understand.....  this is for unit tests, not ui tests, correct?  Is there a way to mock this so we don't need a default value at this point?

Reply:
Updated, Thanks @ear-dev